Marlin/src/pins/stm32/pins_LONGER3D_LK.h

```diff
@@ -96,10 +96,14 @@
 //
 // PWM
 //
-#define SERVO0_PIN         PD13  // Open drain PWM pin on the V0G (GND or floating 5V)
-#define SERVO0_PWM_OD            // Comment this if using PE5
+#if HAS_Z_SERVO_PROBE
```

Comment:
This is the only pins file to define the servo pins conditionally. Note that Marlin won't implement servos unless `NUM_SERVOS > 0` and customarily we always define these pins (which can be useful for M43 pinsDebug).

Reply:
im using this pins file as is with a bltouch

Reply:
`BLTOUCH` helpfully sets `NUM_SERVOS` for you if you forget.